fifeforspark/__main__.py

```diff
@@ -1,4 +1,5 @@
-import processors, utils
+import processors, utils, lgb_modelers
```

Comment:
I think this may actually have to be from fife import lgb_modelers, processors, tf_modelers, utils